Sources/SwiftInspectorCommands/Tests/TypealiasCommandSpec.swift

```diff
@@ -66,6 +66,20 @@ final class TypealiasCommandSpec: QuickSpec {
         it("does not output the typelias identifiers") {
           expect(result?.outputMessage).toNot(contain("SomeType"))
         }
+
+        context("when a typealias is defined multiple times") {
+          it("only returns one the typealias name once") {
+            fileURL = try? Temporary.makeFile(content: """
+                                                       typealias SomeAlias = SomeType
+                                                       typealias SomeAlias = SomethingElse
+                                                       """
+                                                       )
+            result = try? TestTask.run(withArguments: ["typealias", "--path", fileURL!.path])
+
+            let lines = result?.outputMessage?.split { $0.isNewline }.count
+            expect(lines) == 1
```

Comment:
Huh this isn't what I expected. This command can be run on multiple files right? And there is no guarantee that those multiple files are in the same module.

So let's say a file in module A defined `typealias SomeAlias = SomeType` and a file in module B defined `typealias SomeAlias = SomethingElse`. It seems like SwiftInspector _should_ return both.

Reply:
Oh! You're right. So for this case, it shouldn't since this will only output the typealias name, so IMHO we shouldn't repeat the same output and we should only pass it back once (as we do with the imports statements)

You're right that when passing --name we should return both. I'll add that!